style.css

```diff
@@ -35,10 +35,60 @@ header {
   align-content: center;
 }
 
-.navbarMenue {
-  display: none;
+/* mobile menue section */
+@media only screen and (max-width: 768px) {
+  .navbarMenue {
+    visibility: hidden;
+    opacity: 0;
+    width: 100%;
+    height: 100vh;
+    background: rgba(96, 112, 255, 0.95);
+    display: flex;
+    flex-direction: column;
```

Comment:
[OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇 

 - [ ] Nice implementing the design, however, it would be better if you add a little bit of blur to your overlay in order to match the Figma design and also make a good contrast 🎨 👨🏻‍🎨 

- [ ] Make sure the closed menu button (x) stick to the original design as much as possible 🎨 🖌️ 

-  Adding these features will make your design look great and look exactly like the Figma design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this.


| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/79058364/217689963-d1f741d4-1044-46cf-8b6f-46ec423f7a59.png) | ![image](https://user-images.githubusercontent.com/61790587/167505757-7e8c3eba-c166-4f42-842b-829d676cabd5.png) |